src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -32,7 +32,7 @@ import Polygon from "ol/geom/Polygon";
 import HtmlFileReader from "../../../../common/htmlFileReader";
 import { parseTiffData, renderTiffToCanvas, loadImageToCanvas } from "../../../../common/utils";
 import { constants } from "../../../../common/constants";
-import { CanvasCommandBar } from "./CanvasCommandBar";
+import { CanvasCommandBar } from "./canvasCommandBar";
```

Comment:
I get file not found now

Reply:
Windows is case insensitive, so I have to make the change in github online editor. Try sync again. If not work, rename the file manually.